src/EventListener/ControllerListener.php

```diff
@@ -60,6 +60,22 @@ public function onKernelController(KernelEvent $event)
         $classConfigurations = $this->getConfigurations($this->reader->getClassAnnotations($object));
         $methodConfigurations = $this->getConfigurations($this->reader->getMethodAnnotations($method));
 
+        if (method_exists($method, 'getAttributes')) {
+            $classAttributes = $object->getAttributes();
+            array_walk($classAttributes, function (\ReflectionAttribute &$value) {
+                $className = '\\'.$value->getName();
+                $value = new $className($value->getArguments());
+            });
+            $classConfigurations = array_merge($classConfigurations, $this->getConfigurations($classAttributes));
+
+            $methodAttributes = $method->getAttributes();
+            array_walk($methodAttributes, function (\ReflectionAttribute &$value) {
+                $className = '\\'.$value->getName();
+                $value = new $className($value->getArguments());
```

Comment:
A ReflectionAttribute has a `newInstance` property which does the work for us

Reply:
it's so much easier ;)
Thanks for your review @maxhelias !